hack/update-generated-crd-code.sh

```diff
@@ -21,32 +21,4 @@ set -o xtrace
 
 # this script expects to be run from the root of the Velero repo.
 
-if [[ -z "${GOPATH}" ]]; then
-  GOPATH=~/go
-fi
-
-if [[ ! -d "${GOPATH}/src/k8s.io/code-generator" ]]; then
-  echo "k8s.io/code-generator missing from GOPATH"
-  exit 1
-fi
-
-if ! command -v controller-gen > /dev/null; then
-  echo "controller-gen is missing"
-  exit 1
-fi
-
-${GOPATH}/src/k8s.io/code-generator/generate-groups.sh \
-  all \
-  github.com/vmware-tanzu/velero/pkg/generated \
-  github.com/vmware-tanzu/velero/pkg/apis \
-  "velero:v1" \
-  --go-header-file ./hack/boilerplate.go.txt \
-  --output-base ../../.. \
-  $@
-
-controller-gen \
-  crd:crdVersions=v1beta1,preserveUnknownFields=false \
-  output:dir=./pkg/generated/crds/manifests \
-  paths=./pkg/apis/velero/v1/...
-
-go generate ./pkg/generated/crds
+go generate ./config/crd/crds
```

Comment:
I think we need to keep most of the code that was removed here, for the non-kubebuilder API types/clients/etc

Reply:
Hm. 🤔 I think this will work. Here's how I tested it: added a field to `pkg/apis/velero/v1/backup.go` and ran `make update`, and the other 3 files below got updated as expected. If there's nothing missing, then I think it's good like this?

![image](https://user-images.githubusercontent.com/16508/83192457-68d80380-a0ea-11ea-9e71-499407da5205.png)

Reply:
Although all api types (besides the BSL) remain where they were, their corresponding manifest files and the raw CRDs are being generated in the same way and placed in the same places.

Reply:
There's also the generated clients/listers/informers in `pkg/generated`, and the deep-copy functions that go directly in `pkg/apis/velero/v1/zz_generated.deepcopy.go`, that potentially need to update when one of the existing API types updates. I think specifically [this code](https://github.com/vmware-tanzu/velero/pull/2561/files/bf5052d695c4617f97bd62df2d52bfb88d4163ab#diff-892d90adfc5482f1fee4517a4ad27aeeL38-L45) needs to be retained to properly update those.

Reply:
And I guess [this code](https://github.com/vmware-tanzu/velero/pull/2561/files/bf5052d695c4617f97bd62df2d52bfb88d4163ab#diff-892d90adfc5482f1fee4517a4ad27aeeL24-L31) as well.

Reply:
Hmyeah... I totally lost track of this. Will fix.

Reply:
to confirm this works fully, I would suggest deleting all the generated code and then run the code generation step again to confirm everything comes backup.